saleor/discount/models.py

```diff
@@ -56,8 +56,10 @@ class VoucherQueryset(models.QuerySet["Voucher"]):
     def active(self, date):
         subquery = (
             VoucherCode.objects.filter(voucher_id=OuterRef("pk"))
+            .order_by()
+            .values("voucher_id")
             .annotate(total_used=Sum("used"))
-            .values("total_used")[:1]
+            .values("total_used")
```

Comment:
I'm not 100% sure that it will be working when the qs have multiple vouchers, and each voucher has multiple codes with `used`. Test needed for that

Reply:
Tests added